Userland/Libraries/LibVideo/VP9/Decoder.h

```diff
@@ -10,6 +10,9 @@
 #include "Parser.h"
 #include <AK/ByteBuffer.h>
 #include <AK/Error.h>
+#include <LibVideo/DecoderError.h>
+
+#include "Parser.h"
```

Comment:
Uhm, Parser.h is already include a few lines above?

Reply:
Ah, probably goofed up in a rebase. Fixed.